lib/pbench/test/unit/server/auth/test_auth.py

```diff
@@ -289,6 +290,64 @@ def get(self, path: str, **kwargs) -> MockResponse:
 class TestOpenIDClient:
     """Verify the OpenIDClient class."""
 
+    @responses.activate
+    def test_wait_for_oidc_server_fail(self, make_logger):
+        """Verfiy .wait_for_oidc_server() failure mode"""
```

Comment:
s/Verfiy/Verify/ should you make some additional change... repeated in `test_wait_for_oidc_server_succ` at least ...